spec/models/preserved_copy_spec.rb

```diff
@@ -259,4 +259,153 @@
       end
     end
   end
+
+  context '.fixity_check_expired' do
+    let!(:never_checked_pc) { preserved_copy }
+    let!(:checked_before_threshold_pc1) do
+      PreservedCopy.create!(
+        preserved_object_id: preserved_object.id,
+        endpoint_id: endpoint.id,
+        version: 1,
+        status: status,
+        size: 1,
+        last_checksum_validation: (Time.now.utc - 3.weeks)
+      )
+    end
+    let!(:checked_before_threshold_pc2) do
+      PreservedCopy.create!(
+        preserved_object_id: preserved_object.id,
+        endpoint_id: endpoint.id,
+        version: 1,
+        status: status,
+        size: 1,
+        last_checksum_validation: (Time.now.utc - 7.01.days)
+      )
+    end
+    let!(:recently_checked_pc1) do
+      PreservedCopy.create!(
+        preserved_object_id: preserved_object.id,
+        endpoint_id: endpoint.id,
+        version: 1,
+        status: status,
+        size: 1,
+        last_checksum_validation: (Time.now.utc - 6.99.days)
+      )
+    end
+    let!(:recently_checked_pc2) do
+      PreservedCopy.create!(
+        preserved_object_id: preserved_object.id,
+        endpoint_id: endpoint.id,
+        version: 1,
+        status: status,
+        size: 1,
+        last_checksum_validation: (Time.now.utc - 1.day)
+      )
+    end
+    let!(:pcs_ordered_by_query) { PreservedCopy.fixity_check_expired }
+
+    it 'returns PreservedCopies with nils first, then old to new timestamps' do
+      expect(pcs_ordered_by_query).to eq [never_checked_pc, checked_before_threshold_pc1, checked_before_threshold_pc2]
+    end
+    it 'returns no PreservedCopies with timestamps indicating fixity check in the last week' do
```

Comment:
I would like `expect(preservation_policy.ttl_expire).to be a week` to be added to show where the value of "week" comes from ...

Reply:
good call, i thought the 1 week threshold was non-obvious in the context of these tests, and was about to leave a comment, but then that felt like overcommenting.  i like the `expect` approach because it leaves the comment in such a way that the test breaks when it's no longer true 😃

Reply:
i was a bit reluctant to test a settings value or behavior from the PreservationPolicy class in this spec.  so i ended up doing something slightly more clever, but also hopefully more illustrative of where the threshold comes from.